packages/docusaurus-theme-classic/src/theme/Heading/index.js

```diff
@@ -18,7 +18,12 @@ const Heading = Tag => ({id, ...props}) => {
   return (
     <Tag {...props}>
       <a aria-hidden="true" tabIndex="-1" className="anchor" id={id} />
-      <a aria-hidden="true" tabIndex="-1" className="hash-link" href={`#${id}`}>
+      <a
+        aria-hidden="true"
+        tabIndex="-1"
+        className="hash-link"
+        href={`#${id}`}
+        title="Direct link to heading">
```

Comment:
Can we check in react-axe or something if this will cause duplicated title issue on a11y ?

Like every a title will be this `Direct link to heading`

Maybe
```suggestion
        title={`Direct link to heading with id` + id}>
```

Reply:
> Can we check in react-axe or something if this will cause duplicated title issue on a11y ?

No, there will not be such an issue. I checked via the `axe` extension.

Reply:
Ok we're good to merge this then